src/libs/elektra/symbols.map

```diff
@@ -132,6 +132,29 @@ libelektra_0.8 {
 	elektraKeySetName;
 };
 
+libelektra_1.0 {
+	# kdberrors.h
+	ELEKTRA_ERROR_RESOURCE;
```

Comment:
After various frustrating attempts I asked @kodebach despite I wanted to implement it myself. He added those lines

Reply:
I actually didn't add those lines I just moved them from the private part below since they should be accessible by applications. However, you are right they are macros we should be able to remove those lines.